script.js

```diff
@@ -0,0 +1,14 @@
+
+
+
+function webCalculator(){
+    var firstNum, secondNum, sum;
+     firstNum = Number(document.getElementById('firstNum').value);
+     secondNum = Number(document.getElementById('secondNum').value);
+    
+     sum = firstNum + secondNum;
```

Comment:
This is a very readable line. You chose proper variable names and it is very easy to understand what is going on here. You even seperated it from your assignment to the html element which helped make that line more readable.